src/views/Profile/AuthorsList/AuthorsDeletingDialog.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  IconButton,
+  Tooltip,
+} from '@material-ui/core';
+import { DeleteOutlined } from '@material-ui/icons';
+import { useStyles } from './styles/AuthorsTable.styles';
+
+interface IAuthorsDeletingDialogProps {
+  id: number;
+  fullName: string;
+  isAllowedToDelete?: boolean;
+}
+
+const AuthorsDeletingDialog: React.FC<IAuthorsDeletingDialogProps> = (
+  props,
+) => {
+  const { id, fullName, isAllowedToDelete } = props;
+  const classes = useStyles();
+  const [openDialog, setOpenDialog] = useState(false);
+  const [openTooltip, setOpenTooltip] = useState(false);
+
+  const handleChanges = (type: string, value: boolean) => {
+    switch (type) {
+      case 'dialog':
+        setOpenDialog(value);
+        break;
+      case 'tooltip':
+        setOpenTooltip(value);
+        break;
+      default:
+        break;
+    }
+  };
+
+  return (
+    <>
+      <Tooltip
```

Comment:
I think it's better not to merge 2 unrelated component into one. Commented in  AuthorsTableBody component's code